### DISCUSSION TITLE
Anbindung 5575 - 0013 FW 2.14

### DISCUSSION THREAD

**Discussion by andreaseu** [Tom-Bom-badil/samson_trovis_557x#32] [General (de/en)]
Hallo zusammen,

in den nächsten Tagen ziehe ich in ein Haus mit einer Nahwärmesteuerung des Typs Samson Trovis 5575-0013 ein. Nun interessiert mich, ob jemand bereits Erfahrung damit hat, diese Steuerung mithilfe von ESPHome und einem WEMO D1 in Home Assistant zu integrieren? Falls ja, wie sieht die korrekte Verkabelung aus? 

Falls das nicht möglich sein sollte: Gibt es sinnvolle Alternativen, wie den Austausch der Einheit oder den Einsatz smarter Raumthermostate?

Ich würde mich sehr über eure Ratschläge und Erfahrungsberichte freuen. Vielen Dank im Voraus!
Im Anhang ein paar Fotos.
![trovis - 1](https://github.com/user-attachments/assets/8b74775e-e665-4c89-b5bc-ad36f4a0634b)
![trovis - 2](https://github.com/user-attachments/assets/281ce520-0ece-451e-bd2d-5e6d9ec0b7ab)
![trovis - 3](https://github.com/user-attachments/assets/f92ef19b-f698-4b4d-ab56-84030f1ec16b)

**Comment by Tom-Bom-badil** [comment]
Mehrere Jahre waren alle davon überzeugt, dass die 5575 nicht modbusfähig ist - bis @edomifan [hier](https://github.com/Tom-Bom-badil/samson_trovis_557x/discussions/6#discussioncomment-5204224) beschrieben hat, wie es doch geht. Habe es aber selbst nicht ausprobiert, da ich meine 5575 schon sehr bald gegen eine 5576 aus der Bucht getauscht habe.

_WICHTIG: Vorher alle Einstellungen (COx/PAx/Einstellebenen am Drehhregler) aufschreiben, da bei der 5575 alle Einstellungen zurückgesetzt werden müssen. Die Bezeichnung der Einstellungen ist für alle Reglermodelle gleich, daher kannst Du dafür [dieses Excel-File](https://github.com/Tom-Bom-badil/samson_trovis_557x/blob/master/docs) verwenden._

ESPHome-Anbindungen sind im ["Funktionierende Anbindungen" Thread](https://github.com/Tom-Bom-badil/samson_trovis_557x/discussions/6#discussioncomment-7672075) beschrieben, inklusive detaillierter Configs in HA. Das solltest Du 1:1 übernehmen können, da die Registernummern über alle Reglermodelle hinweg gleich sind. Grundlagen zum Verständnis der Anbindung findest Du im [Anhang zum Wiki](https://github.com/Tom-Bom-badil/samson_trovis_557x/wiki/Begriffe-und-'Basics').

Viel Erfolg - würde mich freuen, wenn Du das Ergebnis hier postest! :)

/tom
